.github/CODEOWNERS

```diff
@@ -530,7 +530,7 @@
 /sdk/iot*/                                                         @ethanann-ms @vighatke
 
 # PRLabel: %Iot
-/sdk/modelsrepository/                                             @timtay-microsoft @abhipsaMisra @digimaun @brycewang-microsoft @andyk-ms @tmahmood-microsoft
+/sdk/modelsrepository/                                             @timtay-microsoft @abhipsaMisra @digimaun @brycewang-microsoft @andyk-ms @tmahmood-microsoft @rido-min
```

Comment:
This change will only ensure that you're included as a code reviewer when a PR is opened.   

Did you mean to add yourself to the owners of this package on L536 as well?   That would  ensure that you're notified when issues are opened for support.

Reply:
done

Reply:
@jsquire can you resolve this comment, so I can merge?